app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>List of users</h1>
+<main>
+  <section>
+    <ul>
+      <% @users.each do |user| %>
+        <%= link_to user_user_posts_path(user) do %>
+          <li class="card hover">
+            <div class="user-image border"></div>
+            <div class="user-data border">
```

Comment:
- [ ] I have charged different images for each user, but it is not possible to see them. Please 🙏🏽 use the correct label and `img` and attribute `src` to load the images for a better UI.

![image](https://github.com/MarcoDDM/blog-app/assets/58642949/b8cbe1cb-4346-4164-8ced-b7e1c3924637)